### PR TITLE
Pre-release merge for v0.15.7

### PR DESCRIPTION
**Pull request by mhl10**
# Release notes - arXiv-NG - Version base-0.15.7
                            
<h2>        Bug
</h2>
<ul>
<li>[<a href='https://arxiv-org.atlassian.net/browse/ARXIVNG-2107'>ARXIVNG-2107</a>] -         Race condition during author linking
</li>
<li>[<a href='https://arxiv-org.atlassian.net/browse/ARXIVNG-2109'>ARXIVNG-2109</a>] -         Unwanted urlization in title field of /abs
</li>
<li>[<a href='https://arxiv-org.atlassian.net/browse/ARXIVNG-2114'>ARXIVNG-2114</a>] -         Base does not update the url_map with modified static path
</li>
<li>[<a href='https://arxiv-org.atlassian.net/browse/ARXIVNG-2130'>ARXIVNG-2130</a>] -         &quot;arxiv.org&quot; should not be generically urlized with &quot;this http url&quot;
</li>
</ul>
        
<h2>        Task
</h2>
<ul>
<li>[<a href='https://arxiv-org.atlassian.net/browse/ARXIVNG-2042'>ARXIVNG-2042</a>] -         Retry should be configurable in base Kinesis controller without overriding methods
</li>
<li>[<a href='https://arxiv-org.atlassian.net/browse/ARXIVNG-2100'>ARXIVNG-2100</a>] -         Give middlewares access to the application config
</li>
<li>Add switch extension for Bulma</li>
</ul>

### CODE REVIEW

**Comment by mhl10** [comment]
@erickpeirson you mentioned wanting one last addition to this release; I'll keep this in draft until you're ready

**Comment by erickpeirson** [comment]
@mhl10 Here it is: https://github.com/arXiv/arxiv-base/pull/129/

File: setup.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 setup(
     name='arxiv-base',
-    version='0.15.6',
+    version='0.15.7',
```
Comment (erickpeirson) [inline setup.py]: Maybe I missed it: did the version numbers in ``config.py`` also get updated?

Comment (erickpeirson) [reply setup.py]: Nevermind; now I see it.

**Pull request merged**